pkg/platform/model/svc.go

```diff
@@ -95,6 +96,7 @@ func (m *SvcModel) StopServer() error {
 		return errs.Wrap(err, "Could not create request to quit svc")
 	}
 
+	m.svcm.SetCheckVersion(false)
```

Comment:
My feedback was that this should be internal to the svcm, this is still external.

Reply:
Right, still working on it. Wasn't ready for review. The issue with making this internal the the service manager is that it has no knowledge of a `stop` event. This service manager just starts and waits for the service. I'm looking for a way to give the service manager that knowledge without refactoring the code too much.

Reply:
Oh I'm sorry. I thought it was ready for review.